protocols.csv

```diff
@@ -10,4 +10,5 @@ code	size	name
 421	V	ipfs
 480	0	http
 443	0	https
-444	10	onion
\ No newline at end of file
+477	0	websockets
```

Comment:
Can we please use `V` instead of `0`? As discussed before websocket addresses can have variable length, and this would make it much easier later on